### DISCUSSION TITLE
Need a way for LLMs to know the current date

### DISCUSSION THREAD

**Discussion by ItsNoted** [open-webui/open-webui#3276] [General]
I know this is often ran on private servers not connected to the internet. But how can we tell our LLMs what day it is?

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/releases/tag/v0.3.5 :)

**Reply by ItsNoted** [reply]
> https://github.com/open-webui/open-webui/releases/tag/v0.3.5 :)

Does not really work well with memories though.

**Reply by tjbck** [reply]
Memories are two separate things, I'm struggling to understand why or how current date will be used in memories. Could you elaborate more?

**Reply by ItsNoted** [reply]
To remember things. For example birthdays. If the LLMs cant tell what the current date is they wont know how old someone is.
